src/ui/settings/About.svelte

```diff
@@ -0,0 +1,52 @@
+<script lang="ts">
+    import {getContext} from 'svelte'
+    import {i18nType} from 'src/lib/translations'
+    import {version} from '../../../package.json'
+
+    import {WalletPluginMetadata} from '@wharfkit/session'
+    import BodyText from '../components/BodyText.svelte'
+    import BodyTitle from '../components/BodyTitle.svelte'
+    import Link from '../components/Link.svelte'
+    import {getThemedLogo} from 'src/lib/utils'
+    const {t} = getContext<i18nType>('i18n')
+
+    const wharfLogo = {
+        name: 'Wharf',
+        logo: {
+            light: 'https://raw.githubusercontent.com/wharfkit/assets/master/Logo/Svg/Wharf_logo_bright_horizontal_vector_no_bg.svg',
```

Comment:
We have these on our CDN, and could avoid using Github.

- https://assets.wharfkit.com/wharf-brand-assets/logo/svg/wharf-logo-dark-horizontal-vector-no-bg.svg
- https://assets.wharfkit.com/wharf-brand-assets/logo/svg/wharf-logo-bright-horizontal-vector-no-bg.svg